src/dao/MoodDao.java

```diff
@@ -31,7 +41,41 @@ public MoodDao() {
      * executing the query
      */
     public ArrayList<Mood> getAllMoods() throws DaoException {
-        return null;
+        BufferedReader reader = null;
+        try {
+            ArrayList<Mood> moods = new ArrayList<Mood>();
+            String ln = "";
+
+            reader = new BufferedReader(new FileReader("data/Moods.csv"));
```

Comment:
Let's use a constant here instead of hard-coding the file name in each method if you have time to make the change.